.stylelintrc.js

```diff
@@ -0,0 +1,25 @@
+module.exports = {
+  "extends": [
+    "stylelint-config-standard"
+  ],
+  "customSyntax": "postcss-styled-syntax",
+  "rules": {
+    "no-empty-source": null,
+    "declaration-empty-line-before": null,
+    "alpha-value-notation": "number",
+    "selector-class-pattern": null,
+    "declaration-block-no-redundant-longhand-properties": [
+      true, 
+      { "ignoreShorthands": ["inset"]}
+    ],
```

Comment:
This spots declarations that could be short-handed. Which is good, but sometimes I think it's more readable when split out, up for chats though.